fairing/train.py

```diff
@@ -76,17 +79,14 @@ def compile_ast(self):
             ast, self.repository, self.image_name, self.image_tag, volumes, volume_mounts)
         return ast, env
 
-    def get_metaparticle_client(self):
-        return MetaparticleClient()
-
     def fill_image_name_and_tag(self):
         if self.image_tag is None:
-            self.image_tag = get_unique_tag()
-        
+            os.environ['JH_UNIQUE_RUN_ID'] = get_unique_tag()
+            self.image_tag = "{}-{}".format(os.environ['JUPYTERHUB_USER'], os.environ['JH_UNIQUE_RUN_ID'])
```

Comment:
This will break when not running in JupyterHub?
Keep in mind we also want to support Jupyter notebooks running in standalone mode as well as regular python files.

Reply:
Do you have an example of running regular python files? I thought regular python files would be invoked only from a notebook.
Anyways, I want a name that's easily findable by the user(and JUPYTERHUB_USER seems like a good candidate). I can think about how to abstract it or use it only if set(default to 'fairing' as you had done before)